src/sm/networkmanager/networkmanager.cpp

```diff
@@ -0,0 +1,757 @@
+/*
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "aos/sm/networkmanager.hpp"
+
+#include "log.hpp"
+
+namespace aos {
+namespace sm {
+namespace networkmanager {
+
+constexpr uint64_t cBurstLen              = 12800;
+constexpr auto     cMaxExposedPort        = 2;
+constexpr auto     cAdminChainPrefix      = "INSTANCE_";
+constexpr auto     cInstanceInterfaceName = "eth0";
+constexpr auto     cBridgePrefix          = "br-";
+
+/***********************************************************************************************************************
+ * Public
+ **********************************************************************************************************************/
+
+Error NetworkManager::Init(StorageItf& storage, cni::CNI& cni, TrafficMonitor& netMonitor, const String& workingDir)
+{
+    LOG_DBG() << "Initialize network manager";
+
+    mStorage    = &storage;
+    mCNI        = &cni;
+    mNetMonitor = &netMonitor;
+
+    auto cniDir = FS::JoinPath(workingDir, "cni");
+
+    if (auto err = mCNI->Init(cniDir, cni::cCNIBinariesPath); !err.IsNone()) {
+        return err;
+    }
+
+    mCNINetworkCacheDir = FS::JoinPath(cniDir, "networks");
+
+    return mNetMonitor->Start();
+}
+
+Error NetworkManager::Close()
+{
+    LOG_DBG() << "Close network manager";
+
+    return mNetMonitor->Close();
+}
+
+Error NetworkManager::UpdateNetworks(const Array<aos::NetworkParameters>& networks)
+{
+    LOG_DBG() << "Update networks";
```

Comment:
No implementation for the method.

Reply:
currently, it is ok, that this method is not implemented